CitraRNG/manager7.py

```diff
@@ -136,7 +140,7 @@ def updateSOSFrameCount(self):
             count += 1
 
             # Probably stuck in an infinite loop
-            if count > 100000:
+            if count > 20000:
```

Comment:
What is the reasoning for changing this value?

Reply:
None. I just thought that no one would reach more than 20000 SOS advances. Fixed